torch/distributed/rpc.py

```diff
@@ -73,13 +76,18 @@ def init_rpc(name, backend='pg'):
     use ``pg`` (ProcessGroup) backend, ``init_process_group`` must be invoked
     prior to this method.
 
+    Also to be noted: RPC package only support Python3
```

Comment:
This belongs at the top level module documentation, not here...

Reply:
@ezyang we do not have rpc document in top level distributed module documentation yet. we will add this note when we have rpc document